java/src/apps/startup/Bundle.properties

```diff
@@ -60,3 +60,6 @@ ScriptButtonModel.ScriptNotFound=Cannot create button for script {0}; script not
 TriggerRouteModel.RouteNotDefined=Unable to set route "{0}". Route not defined.
 #Error thrown when startup pause thread is interrupted.
 StartupPauseModel.Interrupted={0} second startup pause interrupted early.
+
+#Remove this string when the class StartupActionFactoryScaffold is removed
+String  = String
```

Comment:
You can write your test without polluting the JMRI sources to do; please remove this.

Reply:
Fixed. Thanks.